src/tests/class_tests/openms/source/TheoreticalSpectrumGenerator_test.cpp

```diff
@@ -591,23 +591,15 @@ START_SECTION(([EXTRA] test isotope clusters for all peak types))
     263.15390,
 
     // 405: POS: 78.54256367545 INT: 0.921514272689819
-    79.04108117545,
     79.04424117545,
```

Comment:
why is there this difference between old / new?

Reply:
This one is because we do p_trim by default now. When computing to a total probability threshold, IsoSpec computes some extra configurations above what's necessary to cover the requested probability, and without p_trim those configurations are reported (the rationale being, well, since we already computed them, we might as well report them). The problem is that exactly which excess configurations are included is undefined, the only guarantee is that sum of reported probabilities is going to be above the selected p. Which means that the exact number of confs of the reported envelope is undefined, and might change as we improve the algorithm (and this number shouldn't be tested for).

I thought that this is a bit counter-intuitive, and now we do p-trim by default. With p-trim we do have the guarantee that the reported set is optimal, that is, it is the smallest set of configurations that sums up to more than the requested probability.

Reply:
And yes, with p-trim the size should not change (much... see below) as we modify the algorithm.